sympy/functions/elementary/complexes.py

```diff
@@ -467,7 +466,8 @@ def eval(cls, arg):
                     return Abs(base)**exponent
                 if base.is_positive == True:
                     return base**re(exponent)
-                return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
+                if base.is_negative:
+                    return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
```

Comment:
After this line we could have `return` alone to take care of the remaining cases where neither `base.is_nonnegative` nor `base.is_negative` is satisfied.